recipe/meta.yaml

```diff
@@ -1,12 +1,13 @@
-{% set version = "0.17.2" %}
+{% set name = "myst-nb" %}
+{% set version = "1.0.0" %}
 
 package:
-  name: myst-nb
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/m/myst-nb/myst-nb-{{ version }}.tar.gz
-  sha256: 0f61386515fab07c73646adca97fff2f69f41e90d313a260217c5bbe419d858b
+  url: https://pypi.io/packages/source/m/myst-nb/myst_nb-{{ version }}.tar.gz
+  sha256: 9077e42a1c6b441ea55078506f83555dda5d6c816ef4930841d71d239e3e0c5e
 
 build:
   number: 0
```

Comment:
let's also add the belt-and-suspenders `--no-build-isolation` to `pip install` to catch build system drift